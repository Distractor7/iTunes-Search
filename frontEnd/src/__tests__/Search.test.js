import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Search from "../components/search";

describe("Search component", () => {
  it("should render correctly and match the snapshot", () => {
    const { asFragment } = render(<Search />);
    expect(asFragment()).toMatchSnapshot();
  });
});
