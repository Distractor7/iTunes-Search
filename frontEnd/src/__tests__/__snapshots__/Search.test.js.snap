// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render correctly and match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="searchBarDiv"
  >
    <form>
      <input
        class="inputForm"
        placeholder="Search term"
        type="text"
        value=""
      />
      <select
        class="media"
      >
        <option
          value="music"
        >
          music
        </option>
        <option
          value="ebook"
        >
          ebook
        </option>
        <option
          value="podcast"
        >
          podcast
        </option>
        <option
          value="movie"
        >
          movie
        </option>
      </select>
      <button
        class="label"
        type="submit"
      >
        Search
      </button>
    </form>
    <div
      class="resContainer"
    />
  </div>
</DocumentFragment>
`;
